frontend/packages/knative-plugin/src/components/services/ServiceRow.tsx

```diff
@@ -58,7 +59,13 @@ const ServiceRow: RowFunction<ServiceKind> = ({ obj, index, key, style }) => {
         {(readyCondition && readyCondition.status) || '-'}
       </TableData>
       <TableData className={tableColumnClasses[7]}>
-        {(readyCondition && readyCondition.message) || '-'}
+        <ClampedText
```

Comment:
Missing usage in `RevisionRow.tsx` as well.

Reply:
updated `RevisionRow.tsx` with `ClampedText`